.bit/responses/2.1-Week 2 Step 1.md

```diff
@@ -0,0 +1,83 @@
+---
+files: n/a
+stepType: checks
+scripts: test.parseimage.js
+week: 2
+step: 1
+name: Week 2 Step 1
+---
+
+## Week 2 Step 1
+
+### [Parsing](https://oxylabs.io/blog/what-is-data-parser#:~:text=Data%20parsing%20is%20a%20method,be%20easily%20read%20and%20understood.) Multipart
+
+In your Azure Portal, go into your HTTP function and into the index.js file. You should see starter code there that has something like `module.exports...`.
```

Comment:
It would be awesome if you could put the content into dropdowns! Here's the syntax for that:

```md
<details>
<summary>:question: Why do we need these "packages" and what are they?</summary>
  </br>

Packages are awesome! They're chunks of publicly available code that someone else has written to help make coding easier for everyone else. These packages reusable code that increases functionality in your code. 

Before the Azure Function can run the code we will write, we have to install all the necessary package dependencies. These packages contain **code that we will "depend on" in the application**; we have to install them in the console using `npm install`.

[What is a package?](https://www.w3schools.com/nodejs/nodejs_npm.asp)


[What is the node-fetch package?](https://www.npmjs.com/package/node-fetch)

[What is the parse-multipart package?](https://www.npmjs.com/package/parse-multipart)
  <br><br/>
</details>
```